Public/template/login_header.php

```diff
@@ -3,5 +3,5 @@
 ?>
     <body class="login">
         <header>
-            <h1 class="login-heading"><img src="<?= IMG_PATH ?>Libertempo143x39.png" alt="Libertempo"></h1>
+            <h1 class="login-heading"><img src="<?= IMG_PATH ?><?= $title ?>143x39.png" alt="<?= $title ?>"></h1>
```

Comment:
Was more a trick to customize homepage logo, but certainly not the best solution.

Reply:
I find it risky, in case of change of name without the file, an error will occur. the ideal would be to import the image from the configuration form... for the time being, leaves "Libertempo", replacing the image without changing the name is easier

Reply:
oops, it's already done :+1: